src/polkit/polkitauthority.c

```diff
@@ -113,9 +114,19 @@ on_proxy_signal (GDBusProxy   *proxy,
                  gpointer      user_data)
 {
   PolkitAuthority *authority = POLKIT_AUTHORITY (user_data);
+  guint16 msg_mask;
+
   if (g_strcmp0 (signal_name, "Changed") == 0)
     {
-      g_signal_emit_by_name (authority, "changed");
+      if ((parameters != NULL) && g_variant_check_format_string(parameters, "(q)", FALSE ) )
```

Comment:
Unnecessary parens around the first condition, trailing whitespace at the end. Also, the` if` doesn't follow the [GNU Coding Style](https://www.gnu.org/prep/standards/html_node/Formatting.html#Formatting).

Reply:
Fixed spaces and indentation of the block. The parentheses allow better readability and dispel any doubts about operator precedence for the reader. If only GNU coding style adopted this...